### PR TITLE
Streamline and bug fix hospital admissions model tutorial

### LINKED ISSUE #273: Streamline `hospital_admissions_model` tutorial?
The `hospital_admissions_model` tutorial is getting a bit long and, with https://github.com/CDCgov/multisignal-epi-inference/pull/267 merged, it takes quite a while for all of the inference to finish. We should consider removing some parts.

### PR DESCRIPTION
**Pull request by dylanhmorris**
This closes #273 and addresses tutorial convergence issues identified after #267 was merged
## Summary of changes
- Fixes model convergence issues by changing the prior on the negative binomial concentration parameter to a LogNormal.
- Corrects typos in the mathematical description / prior specification
- Extends the mathematical model description
- Removes the padding tutorial since it's redundant to do both generic padding and `InitializeInfectionsExponentialGrowth` (which is itself in a sense a form of padding)
- Adds some section headings.
- Removes day of week effects, pending fixes to them, since arguably they should be a separate tutorial in any case.

More can be done but this is already a big set of changes, so I am leaving it here for the time being.

### CODE REVIEW

**Comment by damonbayer** [comment]
Any idea why the original prior for the over-dispersion didn't work well? When implemented it, the prior looked correct and it led to an appropriate posterior predictive.

**Comment by dylanhmorris** [comment]
I am planning to investigate that separately

File: docs/source/tutorials/hospital_admissions_model.qmd [hunk 1]
```diff
@@ -22,33 +22,35 @@ $$
...
+Where $h(t)$ is the observed number of hospital admissions at time $t$, and $H(t)$ is the number of latent hospital admissions at time $t$. The distribution $\text{HospDist}$ is discrete. For this example, we will use a negative binomial distribution with an inferred concentration.
 
 $$
 \begin{align*}
-h(t) & \sim \text{NegativeBinomial}\left(\text{concentration} = 1, \text{mean} = H(t)\right) \\
-H(t) & = \omega(t) p_\mathrm{hosp}(t) \sum_{\tau = 0}^{T_d} d(\tau) I(t-\tau)
+h(t) & \sim \mathrm{NegativeBinomial}\left(\mathrm{mean} = H(t), \mathrm{concentration} = k\right) \\
+H(t) & = p_\mathrm{hosp}(t) \sum_{\tau = 0}^{T_d} d(\tau) I(t-\tau) \\
+\log[p_\mathrm{hosp}(t)] & \sim \mathrm{Normal}(\mu=\log(0.05), \sigma=\log(1.1) \\
+\log(k) & \sim \mathrm{Normal}(\mu=\log(1), \sigma=\log(10))
```
Comment (damonbayer) [inline docs/source/tutorials/hospital_admissions_model.qmd:32]: any particular reason for using $log$ when defining the standard deviation?

File: docs/source/tutorials/hospital_admissions_model.qmd [hunk 2]
```diff
@@ -22,33 +22,35 @@ $$
...
+The reproduction number $\mathcal{R}(t)$ is modeled as a random walk in logarithmic space, i.e.:
 
 $$
 \begin{align*}
-R(t) & = R(t-1) + \epsilon\\
-\log{\epsilon} & \sim \text{Normal}(\mu=0, \sigma=0.1) \\
-R(0) &\sim \text{TruncatedNormal}(\text{loc}=1.2, \text{scale}=0.2, \text{min}=0)
+\log[\mathcal{R}(t)] & = \log[\mathcal{R}(t-1)] + \epsilon\\
+\epsilon & \sim \text{Normal}(\mu=0, \sigma=0.025) \\
+\mathcal{R}(0) &\sim \text{TruncatedNormal}(\text{loc}=1.2, \text{scale}=0.2, \text{min}=0)
```
Comment (damonbayer) [inline docs/source/tutorials/hospital_admissions_model.qmd:53]: Not new in this PR, but I am curious about the choice of `TruncatedNormal`.

File: docs/source/tutorials/hospital_admissions_model.qmd [hunk 3]
```diff
@@ -141,7 +143,7 @@ inf_hosp_int = deterministic.DeterministicPMF(
 )
 
 hosp_rate = metaclass.DistributionalRV(
-    dist=dist.LogNormal(jnp.log(0.05), 0.1),
+    dist=dist.LogNormal(jnp.log(0.05), jnp.log(1.1)),
```
Comment (damonbayer) [inline docs/source/tutorials/hospital_admissions_model.qmd:146]: $\log(1.1) \approx 0.1$ - why the change?

File: docs/source/tutorials/hospital_admissions_model.qmd [hunk 4]
```diff
@@ -166,11 +168,11 @@ latent_inf = latent.Infections()
 I0 = InfectionInitializationProcess(
     "I0_initialization",
     metaclass.DistributionalRV(
-        dist=dist.LogNormal(loc=jnp.log(100), scale=0.5), name="I0"
+        dist=dist.LogNormal(loc=jnp.log(100), scale=jnp.log(1.75)), name="I0"
     ),
     InitializeInfectionsExponentialGrowth(
         gen_int_array.size,
-        deterministic.DeterministicVariable(0.5, name="rate"),
+        deterministic.DeterministicVariable(0.05, name="rate"),
```
Comment (damonbayer) [inline docs/source/tutorials/hospital_admissions_model.qmd:175]: Why the change?

File: docs/source/tutorials/hospital_admissions_model.qmd [hunk 5]
```diff
@@ -166,11 +168,11 @@ latent_inf = latent.Infections()
 I0 = InfectionInitializationProcess(
     "I0_initialization",
     metaclass.DistributionalRV(
-        dist=dist.LogNormal(loc=jnp.log(100), scale=0.5), name="I0"
+        dist=dist.LogNormal(loc=jnp.log(100), scale=jnp.log(1.75)), name="I0"
```
Comment (damonbayer) [inline docs/source/tutorials/hospital_admissions_model.qmd:171]: Why?

File: docs/source/tutorials/hospital_admissions_model.qmd (hunk 1 above)
Comment (dylanhmorris) [reply docs/source/tutorials/hospital_admissions_model.qmd:32]: Makes it easier to compute the implications of the 68/95 rule on the natural scale.

$\exp\Big(\log(M) - \log(S)\Big)$ = M / S$

$\exp\Big(\log(M) + 2 * \log(S)\Big) = MS^2$

etc.

File: docs/source/tutorials/hospital_admissions_model.qmd
Comment (damonbayer) [inline docs/source/tutorials/hospital_admissions_model.qmd:1]: Not changed in this PR, but could we improve the x-axis tick labels for the first figure?
image

**Review by damonbayer — APPROVED** [review]
Looks good. Some minor comments that could be addressed. Some of the removed code here might be useful to "archive" somewhere for later use. For example, the figure we had that used `plot_lm`.

File: docs/source/tutorials/hospital_admissions_model.qmd (hunk 4 above)
Comment (damonbayer) [reply docs/source/tutorials/hospital_admissions_model.qmd:175]: Assume it's just to make the fit better.

File: docs/source/tutorials/hospital_admissions_model.qmd (hunk 2 above)
Comment (dylanhmorris) [reply docs/source/tutorials/hospital_admissions_model.qmd:53]: Will be replaced with a Normal on $\log(\mathcal{R}(t=0))$ in #275

File: docs/source/tutorials/hospital_admissions_model.qmd (hunk 4 above)
Comment (dylanhmorris) [reply docs/source/tutorials/hospital_admissions_model.qmd:175]: More realistic given the actual data we're fitting to we're going to fix a value.

File: docs/source/tutorials/hospital_admissions_model.qmd
Comment (dylanhmorris) [reply docs/source/tutorials/hospital_admissions_model.qmd:1]: Yes, separate issue I think

**Comment by dylanhmorris** [comment]
> Looks good. Some minor comments that could be addressed. Some of the removed code here might be useful to "archive" somewhere for later use. For example, the figure we had that used plot_lm.

I think we can rely on the commit history itself for this.

File: docs/source/tutorials/hospital_admissions_model.qmd (hunk 3 above)
Comment (dylanhmorris) [reply docs/source/tutorials/hospital_admissions_model.qmd:146]: See https://github.com/CDCgov/multisignal-epi-inference/pull/281#discussion_r1684809312

File: docs/source/tutorials/hospital_admissions_model.qmd (hunk 5 above)
Comment (dylanhmorris) [reply docs/source/tutorials/hospital_admissions_model.qmd:171]: See https://github.com/CDCgov/multisignal-epi-inference/pull/281#discussion_r1684809312
